packages/server/lib/controllers/records/getRecords.ts

```diff
@@ -65,7 +66,7 @@ export const getPublicRecords = asyncWrapper<GetPublicRecords>(async (req, res)
 
     const result = await records.getRecords({
         connectionId: connection.id,
-        model: query.model,
+        model: query.variant && query.variant !== 'base' ? `${query.model}::${query.variant}` : query.model,
```

Comment:
using the naming convention `model::variant` as the full model name. It will be enforced by the `runner-sdk` in following PR